nettacker/core/arg_parser.py

```
@@ -708,28 +707,32 @@ def parse_arguments(self):
-                options.passwords = list(set(open(options.passwords_list).read().split("\n")))
+                with open(options.passwords_list) as f:
+                    options.passwords = list(set(f.read().split("\n")))
             except Exception:
                 die_failure(_("error_passwords").format(options.passwords_list))
         # Check custom wordlist
         if options.read_from_file:
             try:
-                open(options.read_from_file).read().split("\n")
+                with open(options.read_from_file) as f:
+                    f.read().split("\n")
             except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)